src/components/ContextChart/index.js

```diff
@@ -134,29 +157,31 @@ class ContextChart extends Component {
 }
 
 export default props => (
-  <Scaler xScalerFactory={createLinearXScale}>
-    <ScalerContext.Consumer>
-      {({
-        xSubDomain,
-        xDomain,
-        updateXSubDomain,
-        contextSeries,
-        xScalerFactory,
-      }) => (
-        <SizeMe monitorWidth>
-          {({ size }) => (
-            <ContextChart
-              width={size.width}
-              {...props}
-              xDomain={xDomain}
-              contextSeries={contextSeries}
-              xSubDomain={xSubDomain}
-              updateXSubDomain={updateXSubDomain}
-              xScalerFactory={xScalerFactory}
-            />
-          )}
-        </SizeMe>
-      )}
-    </ScalerContext.Consumer>
-  </Scaler>
+  <ScalerContext.Consumer>
+    {({
+      domainsByItemId,
+      subDomainsByItemId,
+      timeSubDomain,
+      timeDomain,
+      contextSeries,
+      xScalerFactory,
+      updateDomains,
+    }) => (
+      <SizeMe monitorWidth>
```

Comment:
How does scatterplots deal with dynamic widths/heights?

Reply:
Like line charts, a `<Scatterplot>` will be the height of its container (axes included in the math).